### ISSUE TITLE
Screen Capture but No Windows Visable

### ISSUE THREAD

**Issue by wchorski** [hyperion-project/hyperion.ng#1768]
- [x] I confirm that this is an issue rather than a question.

## Bug report
Screen capture sees desktop but not the windows (i.e. any apps with to display FXs)

### Note
I am using Apple Silicon so this could be a weird compatibility issue. Still noted to allow permissions
<img width="721" alt="image" src="https://github.com/user-attachments/assets/1aa74bf2-94f3-4f3b-8a49-46d1cfc07204">

```shell
MacOs M1 Pro
Ventura 13.6
Hyperion: 2.0.16
```

#### Steps to reproduce
1. Install Hyperion
2. `http://localhost:8090/#conf_grabber`
3. Device discovered = "OSX" (i also tried `QT`)
<img width="624" alt="image" src="https://github.com/user-attachments/assets/8b587a65-d025-4452-b7d2-17fcbe73b354">

4. Actual desktop contains a window
![image](https://github.com/user-attachments/assets/4f5687d9-a3d2-4da3-b314-c71d05447f5c)

#### What is expected?
I expect to see the windows living on the certain destkop (i made sure it wasn't an empty virtual destkop). 

#### What is actually happening?
My output source (wled strips) grab the background desktop image (cyan/white) but nothing else.

#### System

```shell
Hyperion Server:
- Build:             (HEAD detached at 2.0.16) (Paulchen-Panther-cb85d2d/a93d79b-1705568419)
- Build time:        Jan 18 2024 09:19:06
- Git Remote:        https://github.com/hyperion-project/hyperion.ng
- Version:           2.0.16
- UI Lang:           en (BrowserLang: en-US)
- UI Access:         expert
- Avail Screen Cap.: osx,qt
- Avail Video  Cap.: 
- Avail Audio  Cap.: 
- Avail Services:    boblight,effectengine,forwarder,flatbuffer,protobuffer,mDNS,SSDP,borderdetection
- Config path:       /Users/splitskream/.hyperion
- Database:          read/write
- Mode:              GUI

Hyperion Server OS:
- Distribution:      macOS 13.6
- Architecture:      x86_64
- Kernel:            darwin (22.6.0 (WS: 64))
- Root/Admin:        false
- Qt Version:        5.15.12
- Python Version:    3.12.1
- Browser:           Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko)
```

**Comment by Paulchen-Panther** [comment]
I cannot reproduce the problem at the moment. However, I am aware that this phenomenon occurs from time to time.
I just downloaded the installer from GitHub and it worked after I removed Hyperion from the authentication list. After that it asked again if Hyperion is allowed to use screen recording and that worked too.
I'll leave the issue open anyway. Maybe I'll find a solution for it.

Proof:
https://github.com/user-attachments/assets/0af397c4-4452-43ab-813e-cac6017241c2
